rologaki.ino

```diff
@@ -15,12 +15,14 @@ TinyGPSPlus gps; // The TinyGPS++ object for interfacing with the GPS
 
 AltSoftSerial ss; // The serial connection object to the GPS device
 
+// TODO: it's not a password
```

Comment:
This is a password for loging in into the phone. You use it as a command. Be careful here.